portals/cloud-plugins/apip-cloud-ui-insights/README.md

```
@@ -0,0 +1,60 @@
+# APIP Cloud UI — Insights
+
+Shared Moesif Insights embed for **API Control Plane** and **AI Workspace**.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)